src/main/scala/spray/json/CollectionFormats.scala

```diff
@@ -25,7 +27,7 @@ trait CollectionFormats {
   implicit def listFormat[T :JsonFormat] = new RootJsonFormat[List[T]] {
     def write(list: List[T]) = JsArray(list.map(_.toJson).toVector)
     def read(value: JsValue): List[T] = value match {
-      case JsArray(elements) => elements.map(_.convertTo[T])(collection.breakOut)
+      case JsArray(elements) => elements.map(_.convertTo[T]).toList
```

Comment:
`collection.breakOut` removed since Scala 2.13.0-M4

Reply:
This might have performance implications. `breakOut` prevented intermediate collections from being created.